_tools/03_fpki_activity_report.md

```diff
@@ -30,46 +30,47 @@ The following certificates were issued **BY** or **TO** the FPKI Trust Infrastru
+| CertiPath |	CertiPath Bridge CA - G2	| Federal Bridge CA G4	| 3bfc4df881682f8846bff486d422025aee7494d8 | 12/12/2019 |
+| DigiCert	| DigiCert Federated ID L3 CA	| Federal Bridge CA G4	| 28f059e2dffac9de78fb7a1670c082dad2522d3b | 12/16/2019 |
+| DoD	| DoD Interoperability Root CA 2	| Federal Bridge CA G4	| ad9f51f8030956e9b85423256911868fd7a370cb	| 12/16/2019 |
+| Entrust	| Entrust Managed Services NFI Root CA	| Federal Bridge CA G4	| 313f87f0eb4f8e6c658f5e66f58764282c54fba4 | 12/16/2019 |
+| GPO	| GPO PCA	| Federal Bridge CA G4	| de8171556288add44e16d631653c46adab4dcf79 | 12/16/2019 |
+| GSA ACES |	IdenTrust ACES CA 2	| Federal Bridge CA G4	| 1e5a60b592dfdbeea3d99a5225abc5e2239b987e | 12/17/2019 |
+| IdenTrust	| IdenTrust Global Common Root CA 1	| Federal Bridge CA G4	| 2800ea6ecdeb8efc1cf4c042d712e8622e0cbb1a | 12/17/2019 |
+| SAFE	| SAFE Bridge CA 02	| Federal Bridge CA G4	| 600319e6c322229f88e0f434ba96fb0dfd00252e | 12/17/2019 |
+| STRAC	| STRAC Bridge Root Certification Authority	| Federal Bridge CA G4	| c6d4d588000e823cfaf2ecf551ebcd3827fd71b6 | 12/17/2019 |
+| DigiCert	| Symantec Class 3 SSP Intermediate CA - G3	| Federal Bridge CA G4	| a99a0d6510b59b6dc92270eae1f24fbc21d2dcce | 12/18/2019 |
+| TSCP	| TSCP SHA256 Bridge CA	| Federal Bridge CA G4	| db3f9cceb6c6be4c03e0997070d09e7e0e7ee38a | 12/18/2019 |
+| USPTO	| USPTO_INTR_CA1	| Federal Bridge CA G4	9| 78ec2f323452f8f46932b8550663d68b6e96af7 | 12/12/2019 |
 
 
 The following certificates have been removed from the FPKI Trust Infrastructure in the last 30 days.
 
 | Affiliate | Subject CA | Issuing CA | SHA-1 Hash | Expiration Date | Action |
-| --------- | ---------- | ---------- | ------ | ------ |
-| DoD | DoD Interoperability Root CA 1 | SHA-1 Federal Root CA G2 | 369814cdf6813b1cb973a492fdb130d1446296ef | 11/15/2019 | Expire |
-| CertiPath Bridge | CertiPath Bridge CA | SHA-1 Federal Root CA G2 | 71fb94efbcfc8fffaab00db73e92da36a0e601fa | 11/29/2019 | Expire |
-| GSA ACES | ORC ACES 4 | Federal Bridge CA 2016 | 5573fcc5e6ffff2b710181accaa2efdadb8f0f4e | 8/21/2020 | Revoked |
+| --------- | ---------- | ---------- | ------ | ------ | ------ |
+| CertiPath Bridge	| CertiPath Bridge CA - G2	| Federal Bridge CA 2016	| 4d4392ed9abaa619c97dd4ce668937f5d486f1c9	| 12/15/2019 | Expired |
+| USPTO	| USPTO_INTR_CA1	| Federal Bridge CA 2016	| 0704ea9633a45a9a39123bac28be01078c6bfd3a	| 12/15/2019 | Expired |
+| USPTO	| Federal Bridge CA 2016	| USPTO_INTR_CA1	| d8d860c11f216ad1fe07e15caf19b3083c83dbba	| 12/19/2019 | Expired |
+| FPKIMA	| SHA-1 Federal Root CA G2	| Federal Common Policy CA	| 27c589ff2853bd1949cfa433f36a5e285b2e2c7c	| 12/31/2019 | Expired |
+| State	| U.S. Department of State AD Root CA	| Federal Common Policy CA	| c8feb56d0efa3b3b4e2777986304cc19118676a4	| 1/27/2020 | Renewed |
+| Treasury	| US Treasury Root CA	| Federal Common Policy CA	| 5a87922b5eaf1d63198a951b2ab6f59b2f16c131	| 8/29/2021 | Renewed |
```

Comment:
@ryancdickson does this (treasury) belong in the "removed" in the last 30 days section?